lib/pages/main/home_page.dart

```diff
@@ -103,41 +122,28 @@ class _HomePageState extends State<HomePage> {
-                                Material(
-                                  elevation: 5.0,
-                                  borderRadius: BorderRadius.circular(100),
-                                  child: Container(
-                                    height: 60.0,
-                                    decoration: BoxDecoration(
-                                      color: Colors.transparent,
-                                      borderRadius: BorderRadius.circular(100),
-                                    ),
-                                    child: InkWell(
-                                      onTap: () {
-                                        Navigator.push(
-                                          context,
-                                          MaterialPageRoute(builder: (context) => PublicMap()),
-                                        );
-                                      },
-                                      child: Row(
-                                        children: <Widget>[
-                                          Padding(
-                                            padding: EdgeInsets.only(left: 10.0),
-                                            child: SvgPicture.asset('assets/img/maps/public-map-icon.svg', width: 50,),
-                                          ),
-                                          SizedBox(width: 15),
-                                          Expanded(
-                                            child: Text(
-                                              MyLocalizations.of(context, 'public_map_tab'),
-                                              style: TextStyle(
-                                                fontSize: 18,
-                                                color: Colors.grey[600]
-                                              ),
-                                            ),
-                                          ),
-                                        ],
-                                      ),
-                                    ),
+                                Text(
+                                  "Information pills (HC)",
+                                  style: TextStyle(
+                                    fontFamily: "Nunito",
+                                    fontSize: 18.0, // TODO: Change to use TextTheme after the other PR is merged
+                                    fontWeight: FontWeight.w400,
+                                  ),
+                                ),
+                                SingleChildScrollView(
+                                  scrollDirection: Axis.horizontal,
+                                  child: Row(
+                                    children: List.generate(3, (index) {
+                                      return Container(
+                                        margin: EdgeInsets.all(20.0),
+                                        width: 200.0,
```

Comment:
Ensure dynamic value depending on the screen width, so always 1 box and a half appears in the screen